cfme/tests/services/test_myservice.py

```diff
@@ -324,4 +332,39 @@ def test_load_service_with_button():
             6.
             7. Service should load without an error in log
     """
-    pass
+    service, _ = generic_service
+    sd, ele_label = import_dialog
+
+    # Create button group
+    collection = appliance.collections.button_groups
+    button_gp = collection.create(
+        text=fauxfactory.gen_alphanumeric(start="grp_"),
+        hover=fauxfactory.gen_alphanumeric(15, start="grp_hvr_"),
+        type=getattr(collection, "SERVICE"),
+    )
+    request.addfinalizer(button_gp.delete_if_exists)
+
+    # Create custom button under group
+    button = button_gp.buttons.create(
+        text=fauxfactory.gen_alphanumeric(start="btn_"),
+        hover=fauxfactory.gen_alphanumeric(15, start="btn_hvr_"),
+        dialog=sd.label,
+        system="Request",
+        request="InspectMe",
+    )
+    request.addfinalizer(button.delete_if_exists)
+
+    # Navigate to Details page of service
+    view = navigate_to(service, "Details")
+
+    # Start log check
+    log = LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        failure_patterns=[r".*ERROR.*"],
+    )
+    log.start_monitoring()
```

Comment:
Hey @niyazRedhat 
Where are you validating the logs?
Can you use ```with LogValidator``` herer?

Reply:
Yup, need to validate the logs, I agree with @ganeshhubale about the `with LogValidator` suggestion.

Reply:
Yeah right :)